modules/ROOT/pages/outbound-message-routing.adoc

```diff
@@ -1,28 +1,31 @@
 = Outbound Message Routing
 
-The Partner Manager outbound message processing framework enables you to implement receiving endpoints in a variety of ways:
-
-You can:
+The Anypoint Partner Manager outbound message processing framework enables you to implement receiving endpoints in a variety of ways. You can:
```

Comment:
Does "outbound message processing framework" sound a little wordy to you? Is it just another way of saying "routing"?